components/ToggleSwitch.tsx

```diff
@@ -1,9 +1,33 @@
 import styles from "../styles/ToggleSwitch.module.css";
+import { ChangeEventHandler } from "react";
+
+const setDark = () => {
+  localStorage.setItem("theme", "dark");
+
+  document.documentElement.setAttribute("data-theme", "dark");
+};
+
+const setLight = () => {
+  localStorage.setItem("theme", "light");
+  document.documentElement.setAttribute("data-theme", "light");
+};
+
+const toggleTheme: ChangeEventHandler<HTMLInputElement> = (e) => {
+  if (e.target.checked) {
+    setLight();
+  } else {
+    setDark();
+  }
+};
```

Comment:
Hier is op zich helemaal niks mis mee, maar mss is dit cleaner?
```suggestion
const toggleTheme: ChangeEventHandler<HTMLInputElement> = (e) => {
  const targetTheme = e.target.checked ? "light" : "dark";
  localStorage.setItem("theme", targetTheme);
  document.documentElement.setAttribute("data-theme", targetTheme);
};
```

Reply:
U vraagt, wij draaien...